.changeset/a11y-intermediateCheckbox.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-docs': patch
```

Comment:
Because this is a change to the component itself, we want this changeset to be tracked for react-magma-dom:

```suggestion
'react-magma-dom': patch
```

Reply:
I'll edit it.